src/services.jsx

```diff
@@ -0,0 +1,31 @@
+// import axios from 'axios';
```

Comment:
Why did you comment all of this?
In scope you `services.js` you should define your HTTP-Clients. No need to handle custom logic, you can just define here base functions for performing request like I did [here](https://github.com/dotnet-fizzyy/Diploma/blob/master/web/src/api/axiosBaseApi.ts#L3), it will add additional abstraction layer to you.

Reply:
Removed comments and will check your suggestion link later.